server/seed.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require("mongoose")
+const connectDB = require("./db")
+const User = require("./models/User")
+
+require("dotenv").config()
+
+async function seed() {
+    
+    try {     
+        await mongoose.connect(process.env.MONGO_URI, {
+        useNewUrlParser: true,
+        useCreateIndex: true,
+        useUnifiedTopology: true,
+        useFindAndModify: false
+        });
+        
+        await mongoose.connection.db.dropDatabase()
+    
+        await User.create({
+            username: "DemoUser",
+            email: "demo@kanban.com",
+            password: "123456"

```

Comment:
Excellent! Let's go a head and put the username, email, and password in environment vars. Update the sample env as well so other devs know what to input into that information.